ogc/src/main/scala/geotrellis/server/ogc/style/LegendModel.scala

```diff
@@ -0,0 +1,8 @@
+package geotrellis.server.ogc.style
+
+case class LegendModel(
```

Comment:
You can run `headerCreate` and `test:headerCreate` to generate headers, it won't regenerate headers for the files that already have it.